sdk/python/tests/engine/test_cli.py

```diff
@@ -23,7 +23,14 @@ def test_getting_connect_params(fp: FakeProcess):
 @pytest.mark.parametrize(
     "call_kwargs",
     [
-        {"stderr": ["Error: buildkit failed to respond", ""], "returncode": 1},
+        {
+            "stderr": [
+                "Error: buildkit failed to respond.",
+                "If engine was manually stopped, run it again or delete it",
```

Comment:
This is not the expected error message, it's what the engine is returning to the client. Discard these lines.